### PR TITLE
feat: new border-radius tokens

### LINKED ISSUE #2476: Update design-tokens & build script to support new border-radius tokens

### PR DESCRIPTION
**Pull request by Febakke**
Closes #2476

### CODE REVIEW

File: packages/css/chip.css [hunk 1]
```diff
@@ -84,10 +83,18 @@
     outline: none;
     width: var(--dsc-chip-input-size);
 
+    &[type='radio'] {
+      border-radius: 50%;
```
Comment (unekinn) [inline packages/css/chip.css:87]: Denne skal vere `--ds-border-radius-full`, men det har ingen visuell forskjell. Endrer det seinere.

**Comment by mimarz** [comment]
Lets not forget we have to update theme-builder border-radius select to set the new border-radius tokens. 

image

**Comment by Barsnes** [comment]
> Lets not forget we have to update theme-builder border-radius select to set the new border-radius tokens.
> 
> image

I believe this is being changed in #2333. See preview deployment: https://pr-2333.theme.designsystemet.no/

**Comment by Febakke** [comment]
> Lets not forget we have to update theme-builder border-radius select to set the new border-radius tokens.
> 
> image

@Thunear 
When setting up border radius you can choose any value up to `9999`. Maybe we can have a normal textfield with a default value they can change if they need to? 

One issue is that the calculation used to create the border-radius tokens is done inside the token structure. Maybe this should be handled by the CLI instead? Then we can just swap the border-radius values with the Figma plug-in as we do with colors?

**Review by mimarz — APPROVED** [review]

**Pull request merged**